contracts/contracts/lib/ReEncryptionValidator.sol

```diff
@@ -72,64 +73,76 @@ library ReEncryptionValidator {
         //////
 
         // Input validation: E
-        require(checkCompressedPoint(
-            _capsule.pointE.sign,
-            _capsule.pointE.xCoord,
-            _precomputed.pointEyCoord),
+        require(
+            checkCompressedPoint(
+                _capsule.pointE.sign,
+                _capsule.pointE.xCoord,
+                _precomputed.pointEyCoord
+            ),
             "Precomputed Y coordinate of E doesn't correspond to compressed E point"
         );
 
         // Input validation: z*E
-        require(isOnCurve(_precomputed.pointEZxCoord, _precomputed.pointEZyCoord),
-                "Point zE is not a valid EC point"
+        require(
+            isOnCurve(_precomputed.pointEZxCoord, _precomputed.pointEZyCoord),
+            "Point zE is not a valid EC point"
         );
-        require(ecmulVerify(
-            _capsule.pointE.xCoord,         // E_x
-            _precomputed.pointEyCoord,      // E_y
-            _cFrag.proof.bnSig,             // z
-            _precomputed.pointEZxCoord,     // zE_x
-            _precomputed.pointEZyCoord),    // zE_y
+        require(
+            ecmulVerify(
+                _capsule.pointE.xCoord, // E_x
+                _precomputed.pointEyCoord, // E_y
+                _cFrag.proof.bnSig, // z
+                _precomputed.pointEZxCoord, // zE_x
+                _precomputed.pointEZyCoord
+            ), // zE_y
```

Comment:
Minor suggestion where I disagree with the linter.

```suggestion
                _precomputed.pointEZyCoord // zE_y
            ),
```

Some similar instances in later lines in this file.

Reply:
Looks like there is open issue in prettier for that https://github.com/prettier/prettier/issues/807
We can add hack for ignoring this line for prettier or just accept that. Wdyt?